packages/api-client/src/granules.ts

```diff
@@ -411,6 +411,36 @@ export const createGranule = async (params: {
   });
 };
 
+/**
+ * Update granule in cumulus.
+ * PUT /granules
+ * @param {Object} params             - params
+ * @param {Object} [params.body]      - granule to pass the API lambda
+ * @param {Function} params.callback  - async function to invoke the api lambda
+ *                                      that takes a prefix / user payload.  Defaults
+ *                                      to cumulusApiClient.invokeApifunction to invoke the
+ *                                      api lambda
+ * @returns {Promise<Object>}         - the response from the callback
+ */
+export const updateGranule = async (params: {
+  prefix: string,
+  body: ApiGranule,
+  callback?: InvokeApiFunction
+}): Promise<ApiGatewayLambdaHttpProxyResponse> => {
+  const { prefix, body, callback = invokeApi } = params;
+
+  return await callback({
+    prefix,
+    payload: {
+      httpMethod: 'PUT',
+      resource: '/{proxy+}',
+      path: '/granules',
```

Comment:
The path is different from other `update` endpoints, e.g. updateProvider, updateCollection, updateExecution etc., which as `providerId`, `collection name version` or `executionArn` as the parameter.

Reply:
There already exists a `PUT /granules/:granuleName` So I left this as just a plain PUT with the granule in the json data. I thought it was cleaner to leave it out instead of trying to determine which type of put it should be (plus I left it off the POST for create)

Reply:
809a81b1fe